setup/coder-for-docker/postgres.md

```diff
@@ -0,0 +1,4 @@
+---
+title: "External database setup"
+description: Learn how to set up an external Postgres database for use with C4D.
+---
```

Comment:
Should we move the docs from here?
https://coder.com/docs/coder/latest/setup/docker#use-an-external-postgresql-database